content/docs/300-on-brand-menu-api/500-ncr.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: NCR
```

Comment:
```suggestion
title: NCR Aloha
```

It looks like they use "NCR Aloha" on their website so maybe we should do the same?